### PR TITLE
Update PR template for the new changelog workflow.

### PR DESCRIPTION
**Pull request by pengdev**
### Summary of changes

This PR updates PR template for the new changelog workflow.

## Pull request checklist:
 - [x] Briefly describe the changes in this PR.
 - [ ] Include before/after visuals or gifs if this PR includes visual changes.
    
 - [ ] Write tests for all new functionality. If tests were not written, please explain why.
 - [ ] Optimize code for java consumption (`@JvmOverloads`, `@file:JvmName`, etc).
 - [ ] Add example if relevant.
 - [ ] Document any changes to public APIs.
 - [x] Apply changelog label ('breaking change', 'bug :beetle:', 'build', 'docs', 'feature :green_apple:', 'performance :zap:', 'testing :100:') or use the label 'skip changelog'
 - [ ] Add an entry inside this element for inclusion in the `mapbox-maps-android` changelog: `<changelog></changelog>`.
 - [ ] If this PR is a `v10.[version]` release branch fix / enhancement, merge it to `main` firstly and then port to `v10.[version]` release branch.

Fixes: < Link to related issues that will be fixed by this pull request, if they exist >

PRs must be submitted under the terms of our Contributor License Agreement [CLA](https://github.com/mapbox/mapbox-maps-android/blob/main/CONTRIBUTING.md#contributor-license-agreement).

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ If this PR introduces user-facing changes, please note them here.
  - [ ] Add example if relevant.
  - [ ] Document any changes to public APIs.
  - [ ] Apply changelog label ('breaking change', 'bug :beetle:', 'build', 'docs', 'feature :green_apple:', 'performance :zap:', 'testing :100:') or use the label 'skip changelog'
- - [ ] Add an entry inside this element for inclusion in the `mapbox-maps-android` changelog: `<changelog></changelog>`.
+ - [ ] Add an entry inside [CHANGELOG.md](https://github.com/mapbox/mapbox-maps-android/blob/main/CHANGELOG.md) together with this PR, otherwise, apply the 'skip changelog' label.
```
Comment (yunikkk) [inline .github/pull_request_template.md:37]: Hmm, could we remove it completely since CI will anyway fail the build if changelog wasn't modified?

**Review by ank27 — APPROVED** [review]

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ If this PR introduces user-facing changes, please note them here.
  - [ ] Add example if relevant.
  - [ ] Document any changes to public APIs.
  - [ ] Apply changelog label ('breaking change', 'bug :beetle:', 'build', 'docs', 'feature :green_apple:', 'performance :zap:', 'testing :100:') or use the label 'skip changelog'
```
Comment (yunikkk) [inline .github/pull_request_template.md:36]: Also - do we need this step?

File: .github/pull_request_template.md (hunk 1 above)
Comment (pengdev) [reply .github/pull_request_template.md:37]: I guess would be good to add an item here before developer sees the error and fix? I think it's generally a work flow that developers who want to contribute should be aware of.

File: .github/pull_request_template.md (hunk 2 above)
Comment (pengdev) [reply .github/pull_request_template.md:36]: why not? the labels will make the PR more informative.

Comment (pengdev) [reply .github/pull_request_template.md:36]: Maybe we could ask the contributor to categorise their changelog entries based on these tag.

**Review by Chaoba — APPROVED** [review]

**Review by kiryldz — APPROVED** [review]

File: .github/pull_request_template.md (hunk 2 above)
Comment (yunikkk) [reply .github/pull_request_template.md:36]: Yeah but if we add changelog entries manually - why do we need to do it twice? 
We'll need to add this label in the PR AND add the same in the changelog, right?

Comment (pengdev) [reply .github/pull_request_template.md:36]: yes, we should do both, as the label will be useful when we want to search the PR history in the github UI or do categorisation in programmatic way.

While having the category in the changelog makes the changelog more readable.

Comment (yunikkk) [reply .github/pull_request_template.md:36]: If we edit / maintain changelog manually - we can do `git blame` or use Github UI to find the PR that added each changelog entry as well. 


Another question - how will we remember tags for the manual changelog edits? Should we add the whole list somewhere (e.g. on the top of the file) for the reference?

Comment (pengdev) [reply .github/pull_request_template.md:36]: I meant not everyone will have the skills or bother to check the source code/use git blame to check change logs.
As a public repository, adding the label to the PR will also make the PR more descriptive with some metadata that can be easily identified. Which will also allows our customers, PMs or support engineers to use the labels to do some quick search.  I don't see a reason why we don't want to make the PRs as more informative as possible.

> how will we remember tags for the manual changelog edits? Should we add the whole list somewhere (e.g. on the top of the file) for the reference?

Yes, I think we should add some instructions to our documentations, either in DEVELOPING.md, or have a section of description of the categroies in the CHANGELOG.md itself.

**Comment by yunikkk** [comment]
Couple of notes : 
- maps-ios removed the requirements for the labels in the PR's https://github.com/mapbox/mapbox-maps-ios/pull/1066/files and handles everything manually
- common seem to no longer use manual flow, went automatic with https://github.com/mapbox/mapbox-sdk-common/commit/2d483ae25416c001ac4efec522a8e4b2e4552656#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47

@peng I'd suggested to align with maps-ios and remove labeling requirements from PR's.

**Comment by pengdev** [comment]
Closing in favour of https://github.com/mapbox/mapbox-maps-android/pull/1221/files

**Pull request closed without merging**